### PR TITLE
[1186 - ETH-FLOW-V2] Fix and standardise descriptions

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Part of #1186 

Addresses @elena-zh 's comment on descriptions: https://github.com/cowprotocol/cowswap/pull/1187#issuecomment-1277712327

#### Regular Mode
@elena-zh made the comment that the description was a little redundant, so i cleaned it up:
##### Swap Ready base modal text
image

#### Expert Mode
@elena-zh made the point that it is unclear even in expert mode why users are wrapping native currency. I opted for using the ethFlowDescription base text:

##### Approve/Wrap pending
image
image

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @W3stside , great job!
Now texts LGTM and clearly explain about the current flow. 
However, it would be great if @MindyCoW , @anxolin review these texts as well.

One note from my side: still, a user is not warned that currently it is impossible to swap ETH for the case 1 `Native wrapped token is approved AND enough native wrapped token balance for swap` in the Expert mode. Could you please fix it? 

Thanks!

**Review by josojo — COMMENTED** [review]
Looks good to me beside the point that elena mentions

**Review by shoom3301 — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @W3stside , [here ](https://cowservices.slack.com/archives/C036BL4AANP/p1665764406823849?thread_ts=1664361041.864659&cid=C036BL4AANP)is feedback from @MindyCoW . Could you please take a look at it? 

Thanks

**Review by josojo — APPROVED** [review]

File: src/custom/components/swap/EthFlow/pure/EthFlowModalContent/configs.ts [hunk 1]
```diff
@@ -80,24 +86,38 @@ export const ethFlowConfigs: {
...
+          `To continue, click below to wrap your ${nativeSymbol} to ${wrappedSymbol} via an on-chain ERC20 transaction.`,
+        ],
   }),
   [EthFlowState.ApproveNeeded]: ({ isExpertMode, nativeSymbol, wrappedSymbol }) => ({
     title: `Approve ${wrappedSymbol}`,
     buttonText: `Approve ${wrappedSymbol}`,
     descriptions: isExpertMode
-      ? [expertCommonDescription]
+      ? [
+          ethFlowDescription(nativeSymbol),
```
[the remark this answers is not in the document]
Comment (W3stside) [reply src/custom/components/swap/EthFlow/pure/EthFlowModalContent/configs.ts]: for now let's leave it. we would have to check for  previous state which imo is too complex for a 1 line text change

**Review by josojo — APPROVED** [review]

**Comment by W3stside** [comment]
merging without explicit QA approval based on agreement with @josojo and @shoom3301

**Pull request merged**